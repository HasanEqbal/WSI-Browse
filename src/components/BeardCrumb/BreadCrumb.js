import React from 'react';
import './BreadCrumb.scss';

export default function BreadCrumb() {
  return (
    <React.Fragment>
      <ul className="breadcrumbs">
        <li><a href="#">Home</a></li>
        <li><a href="#">Features</a></li>
        <li className="disabled">Gene Splicing</li>
        <li>
          <span className="show-for-sr">Current: </span> Cloning
    </li>
      </ul>
    </React.Fragment>
  )
}
