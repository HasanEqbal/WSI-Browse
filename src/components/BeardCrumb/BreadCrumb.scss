.breadcrumbs{
  margin: auto !important;
  margin-left: 10%;
}

.grid-x{
  width: 79.8%;
}
