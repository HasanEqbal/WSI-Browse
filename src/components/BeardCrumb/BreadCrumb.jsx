import React from 'react';
import './BreadCrumb.scss'

function BreadCrumb({ categoryname }) {
  return (
    <div className="breadcrumb-container">
      <ul className="breadcrumbs grid-x grid-padding-x large-up-3">
        <li><a href="#">Cookware</a></li>
        <li><a href="#">Features</a></li>
        <li className="disabled">Gene Splicing</li>
        <li>
          <span className="show-for-sr">Current: </span> {categoryname}
        </li>
      </ul>
    </div>
  )
}

export default BreadCrumb;
