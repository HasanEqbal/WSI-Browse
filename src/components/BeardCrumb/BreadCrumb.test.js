import React from 'react';
import { shallow } from 'enzyme';
import BreadCrumb from './BreadCrumb';

describe('Component : BreadCrumb', () => {

  it('expect to render BreadCrumb component', () => {
    expect(shallow(<BreadCrumb />)).toMatchSnapshot();
  })

  it('expect to render BreadCrumb component', () => {
    const wrapper = shallow(<BreadCrumb categoryname="New all" />)
    console.log('yyyyyyyy', wrapper.prop)
    expect(wrapper.prop('categoryname')).toEqual('New all')
  })

})




