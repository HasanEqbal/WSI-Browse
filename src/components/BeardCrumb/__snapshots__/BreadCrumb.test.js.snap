// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component : BreadCrumb expect to render BreadCrumb component 1`] = `ShallowWrapper {}`;

exports[`expect to render BreadCrumb component 1`] = `ShallowWrapper {}`;
